src/components/Video/VideoPlayerWidget.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Helmet } from "react-helmet"
+
+export const VideoPlayerWidget = (props) => {
+    useEffect(() => {

```

Comment:
It looks like this library (cloudinary-react) specifies react ^16.3.3 as a peer dependency. If I recall correctly, hooks were introduced in 16.8, so I think that we'd need to update the peer dependency to require ^16.8 or we'd need to re-work this so that it doesn't use hooks.